ui/app/src/components/CreateSiteDialog/BlueprintForm.tsx

```diff
@@ -91,9 +93,9 @@ const messages = defineMessages({
     id: 'createSiteDialog.cantStart',
     defaultMessage: 'Project names may not start with zeros, dashes (-) or underscores (_).'
   },
-  sandboxBranch: {
-    id: 'createSiteDialog.sandboxBranch',
-    defaultMessage: 'Sandbox Branch'
+  gitBranch: {
+    id: 'createSiteDialog.gitBranch',
+    defaultMessage: 'Git Branch'
```

Comment:
There's a couple of message definitions with the same string. Please consolidate and clean up.